src/app/mixins.scss

```diff
@@ -0,0 +1,33 @@
+@function calculateRem($size) {
+  $remSize: $size / 16px;
+  @return $remSize * 1rem;
+}
+
+@mixin font-size($size) {
```

Comment:
@Hariharan482 why this mixin has same property twice

Reply:
removed duplicate property